bigquery/storage_integration_test.go

```diff
@@ -287,10 +300,22 @@ func TestIntegration_StorageReadQueryMorePages(t *testing.T) {
 		Forks NullInt64
 	}
 
+	var firstValue S
+	err = it.Next(&firstValue)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if cmp.Equal(firstValue, S{}) {
```

Comment:
Is it worth amending the backing query to ensure at least one of the columns has a NOT NULL predicate and _some_ value is present?  Or is it sufficient as is because we expect the Null* types to be populated?